README.md

```diff
@@ -1,56 +1,49 @@
 
-#### Editing
-
-The components of the data model are organized in `modules/*`, e.g. in `modules/Assay` we have classes and relations most pertinent to assays: Assay, Assay_Parameter, Platform, annotationProperty (relations).
-So if you wish to add an assay or platform, edit only the relevant file and make a pull request. 
-**Do not edit `NF.csv` or `NF.jsonld` directly!**
-
-For each .csv file, the most important columns are described below; most contributions will need to deal directly only with these, and in a PR a maintainer will help fill the rest. (For advanced contributions, you can DM one of the maintainers.)  
-
+#### Steps to add an attribute to the Metadata Dictionary: 
+1. Create a new [branch](https://github.com/nf-osi/nf-metadata-dictionary/branches) (example: patch/add-attribute)
+2. Go to the csv file where the attribute belongs.  The components of the data model are organized in the folder [modules](https://github.com/nf-osi/nf-metadata-dictionary/tree/main/modules), e.g. in [modules/Assay](https://github.com/nf-osi/nf-metadata-dictionary/tree/main/modules/Assay) we have classes and  most pertinent to assays: Assay, Assay_Parameter, Platform,etc. as well as the relations csv called "annotationProperty". 
+3. Add the attribute to the relevant csv file.  
+4. Complete the columns for the attribute. At a minimum, add the following information - 
 * `Attribute` : (REQUIRED) Name of the new concept.  
 * `Description` : (REQUIRED) A description for the concept.  
-* `Valid Values` : Only required if you are editing an `annotationProperty.csv` file, where you would be defining a new field (relation) and specifying what values are allowed to be used with this field (relation).  
-For example, _tumorType_ is to valid values _{ Malignant Peripheral Sheath Tumor, Schwannoma, ... }_ as _eats_ is to valid values _{ pizza, sandwich, egg, ... }_.    
+* `Valid Values` : Only required if you are editing an `annotationProperty.csv` file, where you would be defining a new field (relation) and specifying what values are allowed to be used with this field (relation). For example, _tumorType_ is to valid values _{ Malignant Peripheral Sheath Tumor, Schwannoma, ... }_ as _eats_ is to valid values _{ pizza, sandwich, egg, ... }_.    
 * `Source` : Preferably a URI to an ontology source term.
+If you need help, you can do a pull request and the maintainer will help fill out the rest.  
+4. If you are adding an attribute that is a valid value for another attribute, navigate to that csv file and add the term.
+5. Create a pull request (PR) and add @allaway or @anngvu as a reviewer. Creating the PR will perform an action to update the [NF.csv](https://github.com/nf-osi/nf-metadata-dictionary/blob/main/NF.csv) file and the [NF.jsonld](https://github.com/nf-osi/nf-metadata-dictionary/blob/main/NF.jsonld) from the modules folder. **You do not need to edit the `NF.csv` or `NF.jsonld` directly because it will be done automatically**
```

Comment:
Why not `add @allaway, @anngvu, or @cconrad8 as a reviewer`. You are a maintainer now! 😁

Reply:
@anngvu thanks for the suggestions! I try to change it so it applies to both the contributor and us (maintainer). 
Let me know your thoughts!